readwise_vector_db/db/database.py

```
@@ -2,66 +1,141 @@
+        AsyncSession: Database session for async operations
+    """
+    if AsyncSessionLocal is None:
+        _initialize_database()
+    
     async with AsyncSessionLocal() as session:
         yield session
+
+
+async def get_pool():
+    """Initialize database pool for pre-warming.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)